ecr-login/api/client.go

```diff
@@ -145,12 +154,12 @@ func (self *defaultClient) getAuthorizationToken(registryID string) (*Auth, erro
 	if err != nil || output == nil {
 		if err == nil {
 			if registryID == "" {
-				err = fmt.Errorf("Mising AuthorizationData in ECR response for default registry")
+				err = fmt.Errorf("ecr: Mising AuthorizationData in ECR response for default registry")
 			} else {
-				err = fmt.Errorf("Missing AuthorizationData in ECR response for %s", registryID)
+				err = fmt.Errorf("ecr: Missing AuthorizationData in ECR response for %s", registryID)
 			}
 		}
-		return nil, err
+		return nil, errors.Wrap(err, "ecr: Failed to get authorization token")
```

Comment:
With this wrapped this way, I think the `ecr: ` prefix should be dropped from the `fmt.Errorf`'s above so we don't get:

`ecr: Failed to get authorization token: ecr: Missing AuthorizationData in ECR response for ...`

Reply:
Done